### PR TITLE
feat: Add status command to machines

### PR DESCRIPTION
**Pull request by shortdiv**
This PR copies the status command from VM and moves it over the machines, since we haven't yet discussed what we want the output for machine status to be, this is an exact replica of what `fly vm status <image-id>` output

terminal screen with output

### CODE REVIEW

**Comment by shortdiv** [comment]
Not sure why tests are failing, I added the command to the docs file which I believe is where the tests are pulling from

File: docstrings/gen.go [hunk 1]
```diff
@@ -485,6 +485,10 @@ to all instances running in a specific region using the --region/-r flag.`,
 		return KeyStrings{"stop <id>", "Stop a Fly machine",
 			`Stop a Fly machine`,
 		}
+	case "machine.status":
```
Comment (rugwirobaker) [inline docstrings/gen.go:488]: make test is deleting your changes to this file cause it's generated from this file:
https://github.com/superfly/flyctl/blob/b117369194768c80f0d31c4303518fcf296b36d5/helpgen/flyctlhelp.toml#L1009-L1040

**Comment by shortdiv** [comment]
🙌

**Comment by jsierles** [comment]
What do we think about trying to pull these commands under the new CLI format?

**Comment by rugwirobaker** [comment]
@jsierles I would be happy about it. After working in the new format it feels slow to work in this one, especially when it comes to rendering things

**Comment by rugwirobaker** [comment]
@shortdiv once this merged this should help you get  a machine status: https://github.com/superfly/flyctl/pull/806

**Comment by shortdiv** [comment]
Yeah I think we should merge this and then work on those updates later @jsierles

**Pull request merged**